services/121-service/src/user/user-role.entity.ts

```diff
@@ -1,13 +1,12 @@
-import { UserRole } from '../user-role.enum';
-import { Entity, PrimaryGeneratedColumn, Column, ManyToMany } from 'typeorm';
-import { UserEntity } from './user.entity';
+import { Entity, Column, ManyToMany, JoinTable, Unique } from 'typeorm';
 import { ProgramAidworkerAssignmentEntity } from '../programs/program-aidworker.entity';
 import { Base121Entity } from '../base.entity';
+import { PermissionEntity } from './permissions.entity';
 
 @Entity('user_role')
 export class UserRoleEntity extends Base121Entity {
-  @Column()
-  public role: UserRole;
+  @Column({ unique: true })
+  public role: string;
```

Comment:
Shouldn't this become `DefaultUserRole` instead of accepting any string? @RubenGeo

Reply:
No, it should also allow custom user defined roles